lib/src/timer.rs

```diff
@@ -143,6 +146,11 @@ impl TimeoutContainer {
         self.duration
     }
 
+    /// format timeout duration
+    pub fn duration_fmt(&self) -> String {
```

Comment:
To me, if there is no issue here, we have to implement `ToString` trait

Reply:
done. implementing `std::fmt::Display` did the trick ;-)